src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsBuildWrapper.java

```diff
@@ -65,21 +65,24 @@
 
 /**
  * Build wrapper that alters the console so that passwords don't get displayed.
- * 
+ *
  * @author Romain Seguy (http://openromain.blogspot.com)
  */
 public final class MaskPasswordsBuildWrapper extends SimpleBuildWrapper {
 
     private final List<VarPasswordPair> varPasswordPairs;
+    private final List<VarMaskRegex> varMaskRegexes;
 
     @DataBoundConstructor
-    public MaskPasswordsBuildWrapper(List<VarPasswordPair> varPasswordPairs) {
+    public MaskPasswordsBuildWrapper(List<VarPasswordPair> varPasswordPairs, List<VarMaskRegex> varMaskRegexes) {

```

Comment:
Ok, will do. I'm about to go in to a meeting, but I'll have an update within the next few hours.